### PR TITLE
Ivysaur 55 courses upload excel

### PR DESCRIPTION
**Pull request by gracekiely**

### CODE REVIEW

**Review by ColbyFoster3 — APPROVED** [review]
I think its fine but the commit history is very difficult to read through. I'm not sure it will not effect the existing code in main.

**Review by camfiander — CHANGES_REQUESTED** [review]
For some reason any files that got moved in the main branch since this branched off are getting recreated here... I figured merging the main branch would fix it but I guess the merge chaos was too much for poor ol' git. I can clean this up but i'm gonna block the merge until I can get the changes in

File: server/prisma/schema.prisma [hunk 1]
```diff
@@ -31,6 +31,14 @@ enum Role {
...
+  id      Int       @id @default(autoincrement())
+  name    String
+  // Many-to-many relation (verify this later)
+  teacher Teacher[]
+  Class   Class[]
+}
+
 model Teacher {
   id        Int         @id @default(autoincrement())
   user      User        @relation(fields: [userId], references: [id])
```
Comment (camfiander) [inline server/prisma/schema.prisma:44]: This is actually a one-to-many relationship (one teachable can have many teachers, but only one teachable per teacher)

I think a field is missing on the `Teacher` model

Don't forget to run `npx prisma format` in the server directory after making your changes

**Review by camfiander — CHANGES_REQUESTED** [review]
I've cleaned up the moved files, but see most recent comment. The prisma migration is currently failing on this branch

**Review by camfiander — APPROVED** [review]
lgtm

**Pull request merged**